descriptors/ModuleDescriptor-template.json

```diff
@@ -800,9 +794,6 @@
         "circulation-storage.cancellation-reasons.item.delete",
         "user-settings.custom-fields.item.post",
         "user-settings.custom-fields.collection.get",
-        "ui-users.editperms",
-        "capability-sets.collection.get",
```

Comment:
this permission "capability-sets.collection.get" is needed for CapabilitiesUserServiceImpl::queryCapabilitySets

Reply:
currently not system user context is used for this call, so yes, this permission is not needed for system user
See https://github.com/folio-org/mod-consortia-keycloak/blob/master/src/main/java/org/folio/consortia/service/impl/TenantServiceImpl.java#L195